src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -566,14 +566,14 @@ CharMap =
 #    3gp      3rd Generation Partnership Project            video/3gpp
 #    3g2a     3rd Generation Partnership Project 2 Audio    audio/3gpp2
 #    3g2      3rd Generation Partnership Project 2          video/3gpp2
-#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/aac
+#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/mp4
```

Comment:
I'll say that your point is...only your.
As i'm not the one that called them AAC, years ago, but perhaps your friend Apple ?
Whatever, most of the renderers identify them as AAC while displaying infos, so for the users it's more easy to understand, remember, IMO.

Anyway, if it was a suggestion for deleting the MIME type, i'll not follow you on that matter.